flocker/control/script.py

```diff
@@ -0,0 +1,40 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script for starting control service server.
+"""
+
+from twisted.python.usage import Options
+from twisted.internet.endpoints import TCP4ServerEndpoint
+
+from .httpapi import create_api_service
+from ..common.script import (
+    flocker_standard_options, FlockerScriptRunner, main_for_service)
+
+
+@flocker_standard_options
+class ControlOptions(Options):
+    """
+    Command line options for ``flocker-control`` cluster management process.
+    """
+    optParameters = [
+        ["port", "p", 4523, "The port to listen on.", int],
+        ]
+
+
+class ControlScript(object):
```

Comment:
I think product has difficulty with calling this the "control service".  They want to call it the "API server".  Nothing necessarily has to change here, just a note that we might need to keep working to convince them to change their minds.